vuepress/docs/_posts/2022-04-20-why-is-blogging-important.md

```diff
@@ -0,0 +1,60 @@
+---
```

Comment:
Everything looks good. Just wondering, no title to the page or is that in the Header Nathan mentioned